config/config.go

```diff
@@ -110,6 +110,7 @@ type (
 		HealthCheck              *HealthCheckConfig             `yaml:"healthCheck"`
 		NamespaceNameTranslation NamespaceNameTranslationConfig `yaml:"namespaceNameTranslation"`
 		Metrics                  *MetricsConfig                 `yaml:"metrics"`
+		FilterNsData             bool                           `yaml:"filterNsData"`
```

Comment:
`EnableUnsafeNsData` -- My thought here is to flip the boolean around so that the default (if not configured) is that we do the filtering.